### PR TITLE
feat: a contributing page

### PR DESCRIPTION
**Pull request by admbtlr**

### CODE REVIEW

**Comment by admbtlr** [comment]
@sarwar @obdulia-losantos can you one of you OK this please (or make any comments)

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1 +1,81 @@
...
+
+If the answer to either of those two questions are "yes", then you're probably dealing with a security issue. Note that even if you answer "no" to both questions, you may still be dealing with a security issue, so if you're unsure, just email us.
+
+When filing an issue, make sure to answer these five questions:
+>
+1. What version of Aether are you using?
+2. What operating system and processor architecture are you using?
+3. What did you do?
+4. What did you expect to see?
+5. What did you see instead?
```
Comment (obdulia-losantos) [inline CONTRIBUTING.md]: Maybe for those questions we want to include a `TEMPLATE_ISSUE.md` file.

Check these ones as an example:
https://github.com/facebook/react/tree/master/.github

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1 +1,81 @@
...
+
+Then sit back and wait. There will probably be discussion about the pull request and, if any changes are needed, we would love to work with you to get your pull request merged into Aether.
+
+Please adhere to the general guideline that you should never force push to a publicly shared branch. Once you have opened your pull request, you should consider your branch publicly shared. Instead of force pushing you can just add incremental commits; this is generally easier on your reviewers. If you need to pick up changes from develop, you can merge develop into your branch. A reviewer might ask you to rebase a long-running pull request in which case force pushing is okay for that request. Note that squashing at the end of the review process should also not be done, that can be done when the pull request is integrated via GitHub.
+
+### Dveloping a custom consumer
```
Comment (obdulia-losantos) [inline CONTRIBUTING.md]: `### Developing a custom consumer`

**Review by obdulia-losantos — CHANGES_REQUESTED** [review]
:writing_hand:

**Review by obdulia-losantos — DISMISSED** [review]
:nerd_face:

**Review by obdulia-losantos — APPROVED** [review]

**Review by obdulia-losantos — APPROVED** [review]

**Pull request merged**